test/modules/ModuleLib/TestResults/EdgeHubRestartMessageResult.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults
+{
+    using System;
+    using System.Net;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Newtonsoft.Json;
+
+    public class EdgeHubRestartMessageResult : MessageTestResult
+    {
+        public EdgeHubRestartMessageResult(
+            string source,
+            DateTime createdAt,
+            string trackingId,
+            string batchId,
+            string sequenceNumber,
+            DateTime edgeHubRestartedTime,
+            HttpStatusCode edgeHubRestartStatusCode,
+            DateTime messageCompletedTime,
+            HttpStatusCode messageCompletedStatusCode,
+            uint restartSequenceNumber)
+            : base(source, createdAt, TestOperationResultType.EdgeHubRestartMessage)
+        {
+            this.TrackingId = trackingId;
+            this.BatchId = batchId;
+            this.SequenceNumber = sequenceNumber;
+            this.EdgeHubRestartedTime = edgeHubRestartedTime;
+            this.EdgeHubRestartStatusCode = edgeHubRestartStatusCode;
+            this.MessageCompletedTime = messageCompletedTime;
+            this.MessageCompletedStatusCode = messageCompletedStatusCode;
+            this.RestartSequenceNumber = Preconditions.CheckNotNull(restartSequenceNumber, nameof(restartSequenceNumber));
+        }
+
+        DateTime EdgeHubRestartedTime { get; set; }
+
+        public HttpStatusCode EdgeHubRestartStatusCode { get; set; }
+
+        public DateTime MessageCompletedTime { get; set; }
+
+        public HttpStatusCode MessageCompletedStatusCode { get; set; }
+
+        public uint RestartSequenceNumber { get; set; }
+
+        public string GetMessageTestResult()
```

Comment:
where do you need this method? can it call GetFormattedResult()

Reply:
It is unused in this PR but I introduced it here for later used in the TRC verification.

Reply:
Please explain how this is used for TRC verification. thanks.

Reply:
I think it will be very clear once it is use in TRC. This problem should fix itself in the next PR.